wordle-web/src/views/AboutView.vue

```diff
@@ -1,17 +1,20 @@
 <template>
-  <v-sheet color="blue" height="170px" class="pt-10 my-5">
-    <v-card color="indigo-darken-3" class="ma-5" elevation="5" rounded="xl">
-      <v-card-title>Hello World</v-card-title>
+  <v-sheet color="blue" class="pt-10 my-5">
```

Comment:
You might want to remove the blue coloring here

Reply:
I fixed that, thank you for pointing it out.